### DISCUSSION TITLE
Regarding the use of AlDente to delay battery aging.

### DISCUSSION THREAD

**Discussion by thirteenkai** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1374] [General]
When the MacBook is not in use, such as at night when resting and the lid is closed, should it be connected to a power source on standby, or is it fine to unplug the power for normal standby? I want to know which situation can best extend the battery life. However, I'm worried that connecting it might damage the computer. My computer is an M4 series.

**Comment by MatthiasKerbl** [comment]
Hi @thirteenkai,

Thank you for your interest in AlDente. In general, there does not seem to be much difference between being plugged in or unplugged during standby overnight. However, the best is most likely unplugging it and turning it off.
Also, M4 MacBooks are not (yet) fully supported. Therefore, bugs might still occur.

Best,
Matthias

**Comment by thirteenkai** [comment]
Thanks
